### DISCUSSION TITLE
No successful connection

### DISCUSSION THREAD

**Discussion by cacahouse** [alandtse/alexa_media_player#1642] [Q&A]
When I try to connect it stays charging and can't do anything else.

The circle goes round and round forever.

 I have Nabu Casa, 2PF Amazon, and little talent for technology that I compensate with eternal patience.

Thanks for your help :)

![alexa](https://user-images.githubusercontent.com/105993106/174263426-ab0e4798-1784-4cdd-ac37-c62b7df6e4d2.png)

**Comment by ibanmda** [comment]
Same issue! Trying to uninstall and install without successfully...

**Comment by Fougues** [comment]
Same issue!

**Comment by mauriziomontanaro** [comment]
Same problem!
